tests/integration/test_feature_schema.py

```diff
@@ -0,0 +1,114 @@
+import pytest
+
+from labelbox import Tool, MediaType
+
+point = Tool(
+    tool=Tool.Type.POINT,
+    name="name",
+    color="#ff0000",
+)
+
+
+def test_deletes_a_feature_schema(client):
```

Comment:
Decided to not use fixtures for tests, because the feature schema class does not have a "deletable" interface. Schema nodes have a "deleted" property, but we can't create such fixtures.